build.gradle

```diff
@@ -38,14 +38,10 @@ subprojects {
     configurations.all {
         resolutionStrategy {
             failOnVersionConflict()
-            // Force the IDEA SDK provided version of these dependencies to avoid classloading issues.
-            force 'org.apache.httpcomponents:httpclient:4.4.1', 'com.google.guava:guava:17.0'
```

Comment:
The force was to allow developers to include those dependencies but automatically changing the version. By removing that line the current result of introducing conflicting versions of libraries is a build failure (because failOnVersionConflict() is still in there)

My main concern (and my reason for writing this CL) was to reduce the total number of dependencies, so we would have less resistance including ourselves as a binary.

Anyway, I think we can 1. Not sure if my sample is doing enough checks though.